convert2rhel/checks.py

```diff
@@ -78,6 +77,32 @@ def check_tainted_kmods():
         )
 
 
+def check_readonly_mounts():
```

Comment:
This function should be called in `perform_pre_checks` (not in main). So we'll have all pre-checks in one place. Does it make sense?

Reply:
Totally! Fixed in a new commit!